kinematics_base_test/src/test_kinematics_plugin.cpp

```diff
@@ -1,3 +1,39 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2015, Southwest Research Institute
```

Comment:
The file was created in 2015 if I recall correctly, I'm not sure if there are any rules on this but I went on date of creation.